build/build_opm_catalog.sh

```diff
@@ -146,50 +144,19 @@ function validate_opm_bundle() {
 }
 
 function build_opm_catalog() {
-    local FROM_INDEX=""
-    local PREV_COMMIT=${PREV_VERSION#*g} # remove versioning and the g commit hash prefix
-    # check if the previous catalog image is available
-    if [ "$(${CONTAINER_ENGINE} pull "${OLM_CATALOG_IMAGE}":"${PREV_COMMIT}" &> /dev/null; echo $?)" -eq 0 ]; then
-        FROM_INDEX="--from-index ${OLM_CATALOG_IMAGE}:${PREV_COMMIT}"
-        log "Index argument is $FROM_INDEX"
-    fi
-
-    log "Creating catalog image $OLM_CATALOG_IMAGE_VERSION using opm"
-
-    ${COMMAND_OPM} index add --bundles "$OLM_BUNDLE_IMAGE_VERSION" \
-                --tag "$OLM_CATALOG_IMAGE_VERSION" \
-                --build-tool "$(basename "$CONTAINER_ENGINE" | awk '{print $1}')" \
-                $FROM_INDEX
-}
-
-function validate_opm_catalog() {
-    log "Checking that catalog we have built returns the correct version $OPERATOR_VERSION"
-
-    local free_port=""
-    local container_id=""
-    local catalog_current_version=""
-
-    free_port=$(python3 -c 'import socket; s=socket.socket(); s.bind(("", 0)); print(s.getsockname()[1]); s.close()')
-
-    log "Running $OLM_CATALOG_IMAGE_VERSION and exposing $free_port"
-    container_id=$(${CONTAINER_ENGINE} run -d -p "$free_port:50051" "$OLM_CATALOG_IMAGE_VERSION")
+    log "Updating the catalog index"
+    ${COMMAND_OPM} render "$OLM_BUNDLE_IMAGE_VERSION" -o yaml  >> olm/deployment-validation-operator-index/catalog.yaml
+    
+    export OPERATOR_NEW_VERSION="deployment-validation-operator.v${OPERATOR_VERSION}"
+    export OPERATOR_PREVIOUS_VERSION="deployment-validation-operator.v${PREV_VERSION}"
```

Comment:
The below `yq` command didn't work for me without exporting those vars :thinking: